zeebe/clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -456,4 +457,24 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    * @return the builder for the command
    */
   DeleteResourceCommandStep1 newDeleteResourceCommand(long resourceKey);
+
+  /**
+   * Command to complete a user task.
+   *
+   * <pre>
+   * long userTaskKey = ..;
+   *
+   * zeebeClient
+   *  .newUserTaskCompleteCommand(userTaskKey)
+   *  .variables(map)
+   *  .send();
+   * </pre>
+   *
+   * <p>If the user task is linked to a process instance then this command will complete the related
```

Comment:
:thinking: Should we also note here that this command only supports communication over REST?

Reply:
Yep, will add it